### DISCUSSION TITLE
[destination-google-sheet] keep the order of the columns

### DISCUSSION THREAD

**Discussion by louise-tt-get** [airbytehq/airbyte#48755] [Connector Ideas and Features]
Hey, 
When exporting data to google sheet, the order of the columns doesn't remain intact. Indeed, instead of exporting a mirror of the table (or datasource), columns are sent in the alphabetical order. 
That brings lots of complexities because another worksheet must be created to reorder the columns as they must be. 
Hightouch manages to preserve the source column order in Google Sheets spreadsheet destination.

Also, no transformation is available with the googlesheet connector, maybe it could be possible to put that feature in the category ?

**Comment by marcosmarxm** [comment]
@nataliekwong, this feature appears from time to time. Are there any plans for a more comprehensive solution to this issue?
@airbytehq/move-destinations can also have some information about limitations.

[a comment by an automated account is not shown]
